test/unit/services/domain/test-capacity-date-range.js

```diff
@@ -37,4 +44,164 @@ describe('services/domain/capacity-date-range', function () {
     expect(capacityDateRange.toFields[1]).to.equal(VALID_TO_MONTH)
     expect(capacityDateRange.toFields[2]).to.equal(VALID_TO_YEAR)
   })
+
+  it('should throw ValidationError if a future date was given for capacity from date', function () {
+    expect(function () {
+      return new CapacityDateRange(
+        VALID_FROM_DAY,
+        VALID_FROM_MONTH,
+        VALID_FUTURE_YEAR,
+        VALID_TO_DAY,
+        VALID_TO_MONTH,
+        VALID_TO_YEAR
+      )
+    }).to.throw(ValidationError)
+  })
+
+  it('should throw ValidationError if a future date was given for capacity to date', function () {
+    expect(function () {
+      return new CapacityDateRange(
+        VALID_FROM_DAY,
+        VALID_FROM_MONTH,
+        VALID_FROM_YEAR,
+        VALID_TO_DAY,
+        VALID_TO_MONTH,
+        VALID_FUTURE_YEAR
+      )
+    }).to.throw(ValidationError)
+  })
+
+  it('should throw ValidationError if capacity from date is more than 6 years old', function () {
+    expect(function () {
+      return new CapacityDateRange(
+        VALID_FROM_DAY,
+        VALID_FROM_MONTH,
+        dateFormatter.now().year() - (CASELOAD_CAPACITY.MAX_HISTORY + 1),
+        VALID_TO_DAY,
+        VALID_TO_MONTH,
+        VALID_FUTURE_YEAR
+      )
+    }).to.throw(ValidationError)
+  })
+
+  it('should throw ValidationError if capacity to date is more than 6 years old', function () {
+    expect(function () {
+      return new CapacityDateRange(
+        VALID_FROM_DAY,
+        VALID_FROM_MONTH,
+        VALID_FROM_YEAR,
+        VALID_TO_DAY,
+        VALID_TO_MONTH,
+        dateFormatter.now().year() - (CASELOAD_CAPACITY.MAX_HISTORY + 1)
+      )
+    }).to.throw(ValidationError)
+  })
+
+  it('should throw ValidationError if given invalid day for capacity from date', function () {
+    expect(function () {
+      return new CapacityDateRange(
+        INVALID_DAY,
+        VALID_FROM_MONTH,
+        VALID_FROM_YEAR,
+        VALID_TO_DAY,
+        VALID_TO_MONTH,
+        VALID_TO_YEAR
+      )
+    }).to.throw(ValidationError)
```

Comment:
Worth verifying that it is the specific validation error message that we expect?